docs/user_manual/source/control_status_registers.rst

```diff
@@ -626,17 +653,17 @@ Detailed:
   is clear an attempt to read the register in user mode will trigger an illegal
   instruction exception.
 
-  +-------+----------+------------------------------------------------------------------+
-  | Bit#  | R/W      | Description                                                      |
-  +=======+==========+==================================================================+
-  | 31:3  | R (0x0)  | Reads return 0x0, writes are ignored.                            |
-  +-------+----------+------------------------------------------------------------------+
-  | 2     | RW       | ``instret`` enable for user mode.                                |
-  +-------+----------+------------------------------------------------------------------+
-  | 1     | R (0x0)  | Reads return 0x0, writes are ignored.                            |
-  +-------+----------+------------------------------------------------------------------+
-  | 0     | RW       | ``cycle`` enable for user mode.                                  |
-  +-------+----------+------------------------------------------------------------------+
+  +-------+------------+------------------------------------------------------------------+
+  | Bit#  | R/W        | Description                                                      |
+  +=======+============+==================================================================+
+  | 31:3  | WARL (0x0) | Reads return 0x0, writes are ignored.                            |
```

Comment:
Search everywhere for 'Reads return 0x0, writes are ignored.' and use 'Hardwired to 0' instead